src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/sge/SGEJobManager.groovy

```diff
@@ -110,4 +82,19 @@ class SGEJobManager extends PBSJobManager {
                 "   1191 0.00000 r140710_09 seqware hqw 07 / 10 / 2014 09:51:48 1",
                 "   1192 0.00000 r140710_09 seqware hqw 07 / 10 / 2014 09:51:48 1")
     }
+
+    @Override
+    void createComputeParameter(ResourceSet resourceSet, LinkedHashMultimap<String, String> parameters) {
+        parameters["-pe"] = "serial ${resourceSet.cores}"
+    }
+
+    @Override
+    void createWalltimeParameter(LinkedHashMultimap<String, String> parameters, ResourceSet resourceSet) {
+        parameters["-l"] = "h_rt=${resourceSet.walltime.toString()}"
+    }
+
+    @Override
+    void createMemoryParameter(LinkedHashMultimap<String, String> parameters, ResourceSet resourceSet) {
+        parameters["-l"] = "h_rss=${resourceSet.getMem().toString(BufferUnit.G)}"
```

Comment:
change to megabyte

Reply:
maybe remove g/m in the toString output (leaking syntax into resource set), but keep the syntax stuff here.

Reply:
I change to MB, but I won't change toString. The class is specifically designed to handle this memory string stuff.

Reply:
Fine.